### DISCUSSION TITLE
Release4 missing ability to update file tags.

### DISCUSSION THREAD

**Discussion by wagnered** [ampache/ampache#2873] [General]
Appears that 4.* had the switches removed from admin/catalog.php that selects `update_file_tags`.  Checked the legacy-develop branch and the switch(es) were still there.

Because the "Update FileTags" select option wasn't removed from the catalog dropdown list, I assume that I discovered a bug. 

If it is a bug, which branch should a 4.* PR be done on to fix it?

**Comment by wagnered** [comment]
Actually, I have to rethink the "update  file tags" process.  Evidently getID3 doesn't limit the number of cover images for Vorbiscomments as it does for  id3 frames.  (And neither did I.)

I'm sure the same problem exists in 5.*.  Should the fix be applied to the 4.* branch or should I just proceed with fixing the current develop branch?

**Comment by usox** [comment]
From what I can see, the option still exists in develop(5)
https://github.com/ampache/ampache/blob/f9d64e0fbb4a754464dfe95d72c8e801b6f9a228/public/admin/catalog.php#L72

Or am I getting this wrong?

**Comment by wagnered** [comment]
You got it right.   Any way, the only problem with develop(5) for me is that AmpConfig::set() has no way to update the configcontainer class.  So there seems to be no way to temporarily disable or enable a config setting.  Ouch!  I just realized that an array can be created and `AmpConfig::set_by_array($array, $clobber)` can be used to update the ConfigContainer. But that seems a little crude.

The problem I referred to above was the problem of correctly writing  pictures to flac files. That probem I created before source-changes existed.. ;'>
